data_science_pipeline/peerscout/api/main.py

```diff
@@ -5,31 +5,41 @@
 from abc import ABC, abstractmethod
 from typing import Optional
 from werkzeug.exceptions import BadRequest, Unauthorized
-
+import jsonschema
+import json
 
 LOGGER = logging.getLogger(__name__)
 
-output1 = {
-    "reviewing_editor_recommendation": {
-        "person_ids": [1, 2, 3],
-        "recommendation_html": "The recommended editors are ..."
-    },
-    "senior_editor_recommendation": {
-        "person_ids": [1, 2, 3],
-        "recommendation_html": "The recommended editors are ..."
-    }
-}
-
-output2 = {
-    "reviewing_editor_recommendation2": {
-        "person_ids": [1, 2, 3],
-        "recommendation_html": "The recommended editors are ..."
-    },
-    "senior_editor_recommendation2": {
-        "person_ids": [1, 2, 3],
-        "recommendation_html": "The recommended editors are ..."
-    }
-}
+REQUEST_JSON_SCHEMA_PATH = os.path.join(os.path.dirname(__file__), 'input-json-schema.json')
+
+
+def response_json(
+        abstract: str,
+        reviewing_editors: list,
+        senior_editors: list
+) -> dict:
+
+    data = {}
+
+    if abstract == "":
+        data['reviewing_editor_recommendation'] = {}
+        data['senior_editor_recommendation'] = {}
+        data['reviewing_editor_recommendation']['person_ids'] = []
+        data['senior_editor_recommendation']['person_ids'] = []
+        data['reviewing_editor_recommendation']['recommendation_html'] = "..."
+        data['senior_editor_recommendation']['recommendation_html'] = "..."
+    elif reviewing_editors == [] and senior_editors == []:
+        data['recommendation'] = "We don't have a recommendation"
+    else:
+        data['reviewing_editor_recommendation'] = {}
+        data['senior_editor_recommendation'] = {}
+        data['reviewing_editor_recommendation']['person_ids'] = reviewing_editors
+        data['senior_editor_recommendation']['person_ids'] = senior_editors
+        data['reviewing_editor_recommendation']['recommendation_html'] = "..."
+        data['senior_editor_recommendation']['recommendation_html'] = "..."
+
+    json_data = json.dumps(data)
```

Comment:
My previous comment regarding the structure is mostly minor.

The one thing I would strongly suggest is to still use `jsonify` and return a dict here (that is also what the function signature suggests). That is to declare the correct response content type.

See https://flask.palletsprojects.com/en/1.1.x/api/?highlight=jsonify

>  It turns the JSON output into a **Response** object with the _application/json_ mimetype.